frontend/src/components/UserTables/DispatcherTable.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState, useEffect } from 'react';
+import DispatcherForm from '../UserForms/DispatcherForm';
+import { Dispatcher } from '../../types/index';
+import styles from './table.module.css';
+
+const DispatcherManager = () => {
+
+  const [dispatchers, setDispatchers] = useState<Dispatcher[]>([]);
+
+  const getCurrentDispatchers = async () => {
+    const dispatchersData = await fetch('/dispatchers')
+      .then((res) => res.json())
+      .then((data) => data.data);
+    setDispatchers(
+      dispatchersData
+    );
+  };
+
+  useEffect(() => {
+    getCurrentDispatchers();
+  }, []);
+
+  const addDispatcher = (newDispatcher: Dispatcher) => {
+    const { id, ...body } = { ...newDispatcher };
+    const requestOptions = {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify(body),
+    };
+    fetch('/dispatchers', requestOptions)
+      .then((res) => {
+        if (res.status !== 200) {
+          throw new Error('adding dispatcher failed');
+        } else {
+          console.log("get error")
```

Comment:
Don't think you need this else branch. Regardless, this is the branch that is good, so you wouldn't print this here